tests/testthat/test-query.R

```diff
@@ -190,3 +190,27 @@ test_that("get_metadata() is cached", {
     
     identical(table, table_2) |> expect_true()
 })
+
+test_that("database_url() expect character ", {
+  DATABASE_URL() |>
+    expect_s3_class("character")
+})
+
+test_that("get_metadata() returns fibrosis data when specify", {
+  fibro_url = DATABASE_URL("fibrosis.0.2.3.parquet")
+  fibro = get_metadata(remote_url = fibro_url) |> 
+    filter(str_like(sample_, "%GSE122960%")) |> as_tibble()
+  expect_true(nrow(fibro) >= 1)
+}) 
+
+test_that("get_metadata() returns cellxgene and fibrosis data together", {
+  databases = c("fibrosis.0.2.3.parquet", "metadata.0.2.3.parquet")
+  metadata = get_metadata(remote_url = DATABASE_URL(databases)) |>
+    filter(str_like(sample_, "%GSE122960%") | str_like(sample_, "bd54ab01e2d9cafabe482e9a1a599780")) |>
```

Comment:
Is it possible to write a query that more obviously demonstrates that both datasets are present? The query you've used here is probably correct, but it uses identifiers that aren't particularly intuitive.

Reply:
read_parquet() has an filename option which would create a column demonstrating file names of both datasets. However, filename column already exists in cellxgene metadata, thus produces an error. 

`Error: Binder Error: Using filename option on file with column named filename is not supported`

An alternative way could be renaming existing filename column in cellxgene

Reply:
Is there a unique cell type in the fibrosis data that you could use to "prove" that it's being included? This would be more comprehensible than an ID in my opinion.